### ISSUE TITLE
cutting using razor tool freezes openshot

### ISSUE THREAD

**Issue by gaining** [OpenShot/openshot-qt#1420]
**System Details:**
* Operating System / Distro: ? Windows 10 x64
* OpenShot Version: ? latest stable, downloaded today
* _Please attach log files if crash_ No crash 👍 

**Issue Description and steps to reproduce:**
Using razor tool and cutting freezes openshot for 30 secs before I'm able to use it again.
System spec is using mobile i7 quad-core with 16GB ram nvidia 840m with 2GB ram. 

For cached data in settings, I gave it 1024MB of memory to use since I have plenty to spare in the hopes that things would be more stable.

I used the old version of openshot half a decade ago, i think v1.4 and although it crashed few times it didn't freeze much on Ubuntu with much less hardware specs. I was wondering if that version is still in development and use Linux to edit my work project instead?

**Comment by peanutbutterandcrackers** [comment]
@gaining - Good sir, could you perhaps please try the latest daily build and see if that works for you? Just head over to the [downloads](https://www.openshot.org/download/) page and click on 'daily builds' and grab the latest one. It has been reported that the latest daily builds work rather well with windows 10.

Hope that helps!

**Comment by gaining** [comment]
You're right, the latest dev branch works much better than the latest stable branch. Razor tool is not as instant as shotcut but it works. A new issue I discovered is that there's no audio at all when playing stuff on the timeline. 
As what I needed to work on was time sensitive because of work, I decided to use shotcut to finish it but I'm glad that you guys are fixing the bugs that makes this program unusable. When I have more time and have a time insensitive project to work on, I'll give openshot another serious try.
